nntrainer/layers/layer_node.h

```diff
@@ -165,6 +165,19 @@ class LayerNode : public ml::train::Layer, public GraphNode {
    */
   unsigned int getNumOutputs() const { return output_layers.size(); }
 
+  /**
+   * @brief Get the number of weights
+   *
+   * @return unsigned int number of weights
+   */
+  unsigned int getNumWeights() const {
+    if (LAYER_V2) {
```

Comment:
Discussed offline to request change this to runtime switch.

sthlike below.
if (layerV1==nullptr) {
}

Reply:
Discussed offline that @zhoonit will take this